### PR TITLE
Link checker action

### PR DESCRIPTION
**Pull request by yceballost**
### :goal_net: What's the goal?
You can trigger this action manually to know if there exist links broken in this repo.

At this moment there is a lot through the markdown documentation
https://github.com/Telefonica/mistica-android/actions/runs/2919732253

### CODE REVIEW

**Comment by gmerinojimenez** [comment]
nice idea, thank you! I didn't thought about this.

Let me discuss with the rest of the team if we can add it to the PR process, so we can't merge a branch with a broken link (and of course, fix the existing ones before)

**Review by gmerinojimenez — APPROVED** [review]
Thank you for the idea and the work! 

feel free to integrate this and we will add it also to the PR process in ANDROID-10988

File: .github/workflows/link_checker.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        uses: lycheeverse/lychee-action@v1.5.1
+        env:
+          GITHUB_TOKEN: ${{secrets.NOVUM_PRIVATE_REPOS}}
+
+      # - name: Create Issue From File
+      #   if: steps.lychee.outputs.exit_code != 0
+      #   uses: peter-evans/create-issue-from-file@v4
+      #   with:
+      #     title: Link Checker Report
+      #     content-filepath: ./lychee/out.md
```
Comment (jeslat) [inline .github/workflows/link_checker.yml:23]: Could you remove this step if it's not necessary?

Comment (yceballost) [reply .github/workflows/link_checker.yml:23]: sure!

**Pull request merged**